dist/ci/travis_script.sh

```diff
@@ -46,15 +39,16 @@ if test -z "$SUBTEST"; then
       pushd ../backend
       make test_unit
       ;;
+    migrate)
+      mv db/structure.sql db/structure.sql.git
+      bundle exec rails db:drop db:create || exit 1
+      xzcat test/dump_2.5.sql.xz | sed '/^USE api_.*;/d' | mysql ci_api_test
```

Comment:
@evanrolfe  Check the compare script for the differences (it was for example the btree stuff, what we did not notice anymore since the removal of the sed line skipped the entire test).

But it is not so important where the deferences where right now, the point is that it would also hide differences in future unnoticed. Eg. when new mysql suddenly changes defaults (again).